### ISSUE TITLE
How do you dump something after the app is loaded?

### ISSUE THREAD

**Issue by ManuelDeLeon** [rwatts3/meteor-packages#6]
For example:
```
  Template.hello.events({
    'click button': function () {
      meteordump(Meteor, { label: "onClick" });
    }
  });
```
(that doesn't do anything)

**Comment by rwatts3** [comment]
Ok i'm looking into this now. 

Quick question keep in mind that Meteor is outside the scope of that template. If you wanted to dump meteor. I would do it on the client but outside of the Template.hello.events. 

>Did you download from atmosphere? The 0.1.0 release is pushed to Atmosphere now. 

If you want to see Meteor Object Dump a few lines lower. 

If you want to see the hello.events object you would do something like this outside of the Template.hello.events Object.
```
meteordump(Template.hello.__eventMaps[0]);
```

**Comment by rwatts3** [comment]
This will return the full event's object. Keep in mind that Meteor Dump, is meant for inspecting the object, less of inspecting the action as it happens within the object. You can typically call meteordump anywhere outside of the event and pick from what happened within the event when it took place.

**Comment by ManuelDeLeon** [comment]
Maybe I don't understand the purpose of this tool. What do you mean 'Meteor is outside the scope of that template'? (Meteor is global and accessible pretty much anywhere) 

I used 'Meteor' just to stay with the original example, but it doesn't work with anything (e.g. `meteordump({ name: 'Frank' });`). btw, it's not just the click event, meteordump only seems to work on bootstrap. For example it doesn't work onRendered either:

```
  Template.hello.onRendered(function(){
    meteordump({ name: 'Frank' });
  });
```

Again, maybe I misunderstand the tool as I thought I could use it to dump objects after the app is loaded, not just when it's bootstrapping.

**Comment by rwatts3** [comment]
If you were to move your meteor dump outside the function does it work ?

**Comment by rwatts3** [comment]
ok you have to pass two arguments. The first argument is the variable you want to dump, everything after that is an option. Name is not a valid argument for the options. It would have to be like this. 

```
meteordump(someObject , {label: 'Frank'});
```

**Comment by ManuelDeLeon** [comment]
Sure.
This works:
```
if (Meteor.isClient) {
  Template.hello.onRendered(function(){

  });
  meteordump({ name: 'Frank' });
}
```
This doesn't:
```
if (Meteor.isClient) {
  Template.hello.onRendered(function(){
    meteordump({ name: 'Frank' });
  });
}
```

**Comment by ManuelDeLeon** [comment]
> you have to pass two arguments

The arguments have no effect on this.

**Comment by rwatts3** [comment]
Oh I see you were passing name:'Frank' as the actual object.

**Comment by rwatts3** [comment]
What do you mean no affect not following sorry

**Comment by rwatts3** [comment]
When I dump this 

`meteordump(Meteor , {label:'Frank'});`

This is the result. 
http://screencast.com/t/Ib0h4HfI4

**Comment by ManuelDeLeon** [comment]
Yes, it works when meteordump is put on the boostrap path, it doesn't work if you do:
```
if (Meteor.isClient) {
  Template.hello.onRendered(function(){
    meteordump(Meteor, { label: 'Frank' });
  });
}
```

**Comment by rwatts3** [comment]
Hmm, 

Ok i'd be happy to take PRs to correct this what are your thoughts ?

**Comment by ManuelDeLeon** [comment]
Unfortunately I don't have to time to work on another project. It would be an empty promise if I said "I'll take a loot at it".
Sorry.

**Comment by rwatts3** [comment]
No worries I truly understand.  In the meantime I'll look into this issue ,
and see about a possible fix in 0.1.1
